test/testUtils/mayaUtils.py

```diff
@@ -250,43 +247,6 @@ def createSingleSphereMayaScene(directory=None):
     cmds.file(save=True, force=True, type='mayaAscii')
     return tempMayaFile
 
-def previewReleaseVersion():
```

Comment:
Completely removed this helper - its currently not used. We have other methods for detecting how to check for things in tests (ufe support fix level, etc).